metrics/lint/aip-linterv2.go

```diff
@@ -21,6 +21,8 @@ import (
 	pb "github.com/googleapis/gnostic/openapiv2"
 )
 
+// Fill Porot takes a slice of rules and coverts them to a slice of
```

Comment:
nit: typo "Porot" - also, I believe the first word of this comment should be the exact function name.